src/main/java/com/crowdin/cli/commands/actions/StatusAction.java

```diff
@@ -92,6 +95,15 @@ public void act(Outputter out, ProjectProperties pb, ProjectClient client) {
                 progresses.forEach(pr -> out.println(String.format(RESOURCE_BUNDLE.getString("message.item_list_with_percents"),
                     pr.getLanguageId(), pr.getApprovalProgress())));
             }
+            if (failedIfIncomplete) {
+                progresses.stream()
+                    .filter(p -> p.getApprovalProgress() <= 100)
+                    .forEach(throwException(RESOURCE_BUNDLE.getString("error.execution_contains_errors")));
```

Comment:
Let's create a separate message for this case: `The current project is incomplete`